app/presenters/provider/admin/backend_apis_index_presenter.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Provider::Admin::BackendApisIndexPresenter
+  include System::UrlHelpers.system_url_helpers
+
+  def initialize(current_account:, params: {})
+    @current_account = current_account
+    @pagination_params = { page: params[:page] || 1, per_page: params[:per_page] || 20 }
+    @search = ThreeScale::Search.new(params[:search] || params)
+  end
+
+  attr_reader :current_account, :pagination_params, :search
+
+  def data
+    {
+      'new-backend-path': new_provider_admin_backend_api_path,
+      backends: page_backend_apis.decorate.map(&:index_page_data).to_json,
+      'backends-count': page_backend_apis.total_entries.to_json
+    }
+  end
+
+  protected
+
+  # TODO: is .order needed?
```

Comment:
technically needed or is the question about whether we want it?

Reply:
Technically speaking.

Reply:
If we want them in this order, then I guess it is needed. My understanding is that we can't guarantee api columns are stored in the order of `updated_at`.

Reply:
I just tested it locally and this is the query for `current_account.backend_apis.reorder(updated_at: :asc)`:
```sql
SELECT `backend_apis`.* FROM `backend_apis`
WHERE `backend_apis`.`account_id` = 2
ORDER BY `backend_apis`.`updated_at` ASC
```
Which I guess is correct since ORDER BY looks correct.

Reply:
Didn't you need descending order? i.e. last updated to show up first?